frontend/lib/shared/utils/number_pad_input_validation_util.dart

```diff
@@ -0,0 +1,26 @@
+class NumberPadInputValidationUtil {
+  static bool validateKeyPress(String input, String key) {
```

Comment:
`.` as the first key press results in an error but will do a fast follow to address this in https://github.com/TBD54566975/didpay/pull/24